src/ParseConfig.js

```diff
@@ -158,6 +175,37 @@ var DefaultController = {
         return config;
       });
     });
+  },
+
+  save(attrs,params) {
+    var RESTController = CoreManager.getRESTController();
+    if(!params || !params.useMasterKey){
+      const error = new ParseError(
+        ParseError.OTHER_CAUSE,
+        'Master Key needed for update Parse.Config.'
+      );
+      return Promise.reject(error);
+    }
+    const encodedAttrs = {};
+    for(const key in attrs){
+      encodedAttrs[key] = encode(attrs[key])
+    }
+    return RESTController.request(
+      'PUT',
+      'config',
+      { params: encodedAttrs },
+      { useMasterKey: true }
```

Comment:
As the masterKey is forced here, I am not sure we should have the Params check above. If the masterKey is not provided at initialization the save will fail anyway

Reply:
It's just for an UX purpose and an integrity reason about the other `.save()` method.
To show an error during a save without the master can help other developers to understand how it works and then adjust their code.
It seems good for me !

Reply:
In this case, without the masterKey set in the SDK it will never work. So this is quite some overhead to write useMasterKey: true everywhere without providing anything in return